ui/src/app/agents/new/page.tsx

```diff
@@ -103,7 +103,7 @@ function AgentPageContent({ isEditMode, agentId }: AgentPageContentProps) {
                 model: agentResponse.model,
                 name: agent.spec.modelConfig,
               });
-              
```

Comment:
looks like there are only formatting changes here; can you revert this? (feel free to make the formatting change in a separate PR if needed).